04_functions/04_special_calls/exercise1.r

```diff
@@ -1,23 +1,10 @@
 ### Create a list of all the replacement functions found in the base package. Which ones are primitive functions?
 
-# This is kind of ugly, but it works:
-  objs <- mget(ls("package:base"), inherits = TRUE)
-  funs <- Filter(is.function, objs)
+# Search for the pattern <- in the base package and bring the function's body(mget). The output is a list.
+  infix <- mget(grep("<-",ls('package:base'), value = T), inherits = T)
+  m <- sapply(infix, is.primitive) # Loop through the list and test which is a primitive and give back a named vector of T's or F's
```

Comment:
Yeah, that would be better... we've learned a lot since this was originally written. :+1: